cmd/check_perf.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	t "github.com/coinbase/rosetta-cli/pkg/tester"
+	"github.com/spf13/cobra"
+)
+
+var (
+	checkPerfCmd = &cobra.Command{
+		Use:   "check:perf",
+		Short: "Benchmark performance of time-critical endpoints of Asset Issuer's Rosetta Implementation",
+		Long: `This command can be used to benchmark the performance of time critical methods for a rosetta server.
+This is useful for ensuring that there are no performance degradations in the rosetta-server.`,
+		RunE: runCheckPerfCmd,
+	}
+)
+
+func runCheckPerfCmd(_ *cobra.Command, _ []string) error {
+	ctx, cancel := context.WithCancel(Context)
+
+	timeTaken := t.Bmark_Block(ctx, cancel, Config, 3000)
```

Comment:
can this be a const somewhere. Let's try to avoid magic numbers. Also it could be a command line argument too. We can start with a const and improve on it